### DISCUSSION TITLE
⏳ 🔗 Loop Grid Search Filter

### DISCUSSION THREAD

**Discussion by nolderoos** [elementor/elementor#27664] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Larger websites have a ton of different post types, loop grids etc. I believe the addition of connecting a search form to a loop grid and have it search that loop grid on that page without load would be amazing!

### Describe the solution you'd like.

Being able to add a search form to connect to a loop grid. Kinda like the current filter function.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by KarBlownForceNet** [comment]
This is exactly what I need!
